frontend/src/editor/output/JsonOutput.tsx

```diff
@@ -77,7 +77,7 @@ function inferBestFormat(data: unknown): "tree" | "raw" {
 const LEAF_RENDERERS = {
   "image/": (value: string) => <ImageOutput src={value} />,
   "video/": (value: string) => <VideoOutput src={value} />,
-  "text/html": (value: string) => <HtmlOutput html={value} inline={true} />,
+  "text/html": (value: string) => <HtmlOutput html={value} />,
```

Comment:
something to watch out for, i removed the inline so charts in a json would expand fully, but this may break other things. easy to revert, because people may not even want to put charts in a json